scipy/stats/stats.py

```diff
@@ -2760,12 +2767,18 @@ def median_absolute_deviation(x, axis=0, center=np.median, scale=1.4826,
         the MAD over the entire array.
     center : callable, optional
         A function that will return the central value. The default is to use
-        np.median. Any user defined function used will need to have the function
-        signature ``func(arr, axis)``.
-    scale : int, optional
-        The scaling factor applied to the MAD. The default scale (1.4826)
-        ensures consistency with the standard deviation for normally distributed
-        data.
+        np.median. Any user defined function used will need to have the
+        function signature ``func(arr, axis)``.
+    scale : scalar or str, optional
+        The numerical value of scale will be divided out of the final
+        result. The following string value is also recognized (default is 1.0):
+
+          * 'normal' : Scale by :math:`\frac{1}{\sqrt{2}erf^{-1}(1/2)}`.
```

Comment:
I would unindent this by two spaces, otherwise it will appear as a bullet inside a block quote instead of just a bullet

Reply:
done